source/quickstart.txt

```diff
@@ -25,15 +24,24 @@ the rest of the {+c2c-product-name+} documentation.
 Setup
 -----
 
-{+c2c-product-name+} syncs data between two clusters. To run
-``mongosync``, you will need to: 
+{+c2c-product-name+} syncs data between two clusters. 
+
+- Each cluster can be either a replica set or a sharded cluster.
+
+- Sharded clusters do not require the same number of shards.
```

Comment:
[suggestion] This is a little difficult to parse -- consider something like, 

"Sharded clusters are not required to have the same number of shards."

Reply:
Fixed